resources/default_pipeline_environment.yml

```diff
@@ -315,3 +315,13 @@ steps:
       developmentSystemId: null
   transportRequestUploadFile:
   transportRequestRelease:
+
+  #defaults for stage wrapper
+  piperStageWrapper:
+    extensionLocation: '.pipeline/extensions/'
+    globalExtensionLocation: '.pipeline/globalExtensions'
+    stageLocking: true
+    nodeLabel: ''
+    stashContent:
+      - 'pipelineConfigAndTests'
```

Comment:
The current stashing concept is not compatible with ours. We have an approach of stashing and unstashing on stage level where you can define for each stage (depending on the name) what is stashed and unstashed. Furthermore, stashes are merged if needed in case stages are executed in parallel.